raster/src/main/scala/geotrellis/raster/histogram/Histogram.scala

```diff
@@ -136,12 +134,12 @@ abstract trait Histogram extends Serializable {
       var needMedian = true
       val limit = getTotalCount() / 2
 
-      var i = 0
       val len = values.length
 
-      while (i < len) {
+      cfor(0)(_ < len, _ + 1) { i =>
```

Comment:
Good to know. I had looked into it and figured out that cfor was basically just being compiled into a while loop but figured that the consistency may be worth it. Will bench this and post something on this PR.